package_manifest.yaml

```diff
@@ -62,6 +62,8 @@ copr_projects:
             - "https://yum.puppet.com/puppet7/el/{{ rhel_8 }}/x86_64/"
             - "{{ foreman_staging }}/rhel-{{ rhel_8 }}-x86_64"
           comps_file: "{{ inventory_dir }}/comps/comps-foreman-el{{ rhel_8 }}.xml"
+          buildroot_packages:
+            - foreman-build
```

Comment:
should this be `core_buildroot_packages` like we have `core_modules` so it's one place to update?

Reply:
I had this feeling...

Reply:
You know me too well…